src/puppetlabs/puppetdb/utils.clj

```diff
@@ -13,7 +13,8 @@
             [slingshot.slingshot :refer [try+ throw+]]
             [com.rpl.specter :as sp])
   (:import [java.net MalformedURLException URISyntaxException URL]
-           [org.postgresql.util PGobject]))
+           [org.postgresql.util PGobject]
+           [puppetlabs.puppetdb.cheshire RawJsonString]))
```

Comment:
I think the spaces in these imports are unnecessary, `[puppetlabs.puppetdb.cheshire RawJsonString]` means import `puppetlabs.puppetdb.cheshire` and `puppetlabs.puppetdb.cheshire.RawJsonString`, I think you just want `[puppetlabs.puppetdb.cheshire.RawJsonString]`.